README.md

```diff
@@ -150,6 +177,19 @@ Taking the above into account, the following combinations for `install.yaml` are
 
 Both ROS distro specific as default can be 'null'. Prevered usage is default for current and feature distributions and exceptions for old distributions.
 
+#### Catkin package install
+
+```yaml
+- type: catkin
+  source:
+    type: git
+    url: <Repository URL>
+    sub-dir: <Sub directory of the repository> (Optional field)
+    version: <Version to be installed> (Optional field)
+```
+This target type is similar to a ROS source target with only difference being a catkin package is independent of any ROS
+dependencies and solely depends on catkin.
```

Comment:
How does this differ in the install?

Reply:
There is no system install possible for this. In install it does not differ a lot, it uses the tue-install-ros command. Plus the target name does not need to start with ros-

Reply:
I think we have a lot of catkin targets right now, which start with `ros`-. As a dependency on ros(rospy/rospp) is not a special case for the installer. What is important is that a catkin WS needs to be available, with the required tools.